config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/MoronsGuidetoBet-AAAAAAAAAAAAAAAAAAAEWw==.json

```diff
@@ -31,7 +31,7 @@
Height§r: controls how high the tree grows. Giant Sequoia is the best, though it may be a little §otoo§r large.\n\nAllele    Value\nSmallest  25%\nSmaller   50%\nSmall      75%\nAverage 100%\nLarge    125%\nLarger   150%\nLargest  175%\nGigantic  BIG\n\n§bGirth§r: the width of your tree.  Wider trees have more leaves and logs, but will also need more saplings to begin growing. All saplings must be the same species and girth. The sapling in the NW corner determines the stats of the new tree, and is where bone meal must be applied to mature the tree. The Giant Sequoia is best here too.\n\nAllele/Size\n1x1\n2x2\n3x3\n4x4 \n\nYield: chances of getting a fruit from a destroyed leaf. Orchard Apple and Chili Pepper are the best here.\n\nAllele     Drop Chance\nLowest   2.5%\nLower    5%\nLow       10%\nAverage 20%\nHigh       30% \nHigher    ?%\n\n§6Sappiness§r: the amount of biomass a sapling produces in the Forestry Fermenter. You can boost the Fermenter\u0027s output by using honey or fruit juice. Blue Mahoe and Red Maple are the best. NEI doesn\u0027t show recipes for analyzed saplings, and probably hybrids too, in the fermenter.§r\n\nAllele  Forestry Biomass\nLowest   100mB\nLower    200mB\nLow       300mB\nAverage 400mB\nHigh       600mB \n\n[warn]Fireproof[/warn]: applies to the wood. Only the Coast and Giant Sequoia have this.\n\n§7Effect§r: not used in trees, so it\u0027s None for any.\n\nBelow is more information on some other parameters:\n\n§2Growth§r: Sipiri trees have Tropical instead of Light, can only be bred in a Biome with temperature 90%-190% and humidity 90%-200%, and can only grow in 120%-190% temp and 90%-200% humidity. All other trees just need light to grow.\n\n[note]Territory/Soil[/note]: all trees have Average, whatever that means. Probably something related to the below that never really got used.\n\n§eTolerates§r: if it can be planted on sand or not. Only Desert Acacia and Grandidier\u0027s Baobab have this. The trait is either Plains or Plains, Desert.\n\n§5Supports§r: what kind of fruits it can theoretically have. Determined by Species. Only fruits from families listed under this trait can be produced by the tree, regardless of what appears under the tree\u0027s Fruits trait. There is a number of trees from the Extra Trees mod that support all 4 types, but none from Forestry itself.\n\n§aFruit§r: determines what fruit or nut it produces. Each fruit/nut provides different advantages:\nPlum and Cherry Plum only produces 100L Fruit Juice, but have a 60% chance of producing Mulch in the Fluid Extractor or Squeezer.\nPapaya produces 600L Fruit Juice, but only has a 10% chance to produce Mulch.\nCoconuts produce 300L Seed Oil, but only have a 25% chance to produce Mulch.\nUse a Forestry Multifarm so you don\u0027t have to break the leaf block to get the fruit/nut. Set it to Manual Orchard, and place everything first.\n\n§bFamily§r: Tells you what family of the fruit listed under the Fruits trait is: Pomes, Prunes, Nuts or Jungle."
```

Comment:
I applied the tags when the colors were specifically the ones used for warning or note, since my objective is to remove as much invisible formatting as possible. Since the colors are the same, there should be no problem.

Reply:
§c is red and §4 is dark red, they are the two colors used for the text in warn tags. The usage of two colors per tag is to ensure text is easy to read between light and dark backgrounds, so I changed those two to warn tags in order to enforce this.

Reply:
I see, thanks for the explanation!